platform/overlays/aks/flux.yaml

```diff
@@ -69,18 +67,27 @@ spec:
           medium: Memory
       containers:
       - name: flux
-        image: docker.io/weaveworks/flux:1.13.0
+        image: docker.io/fluxcd/flux:1.15.0
         imagePullPolicy: IfNotPresent
         resources:
           requests:
             cpu: 50m
             memory: 64Mi
         ports:
         - containerPort: 3030 # informational
+        livenessProbe:
+          httpGet:
+            port: 3030
+            path: /api/flux/v6/identity.pub
+          initialDelaySeconds: 5
+          timeoutSeconds: 5
+        readinessProbe:
+          httpGet:
+            port: 3030
+            path: /api/flux/v6/identity.pub
+          initialDelaySeconds: 5
+          timeoutSeconds: 5
         volumeMounts:
-        - name: acr-auth
```

Comment:
What was this for?

Reply:
This was part of the authentication stuff for Azure Container Registry. It doesn't apply for the moment so I deleted it. Will revisit soon.